src/main/kotlin/io/github/detekt/gradle/extensions/DetektExtension.kt

```diff
@@ -9,6 +9,15 @@ import java.io.File
 open class DetektExtension(project: Project) : CodeQualityExtension() {
 
     var isEnabled: Boolean = true
+    var baseline: File? = null
+    var debug: Boolean = false
+    var parallel: Boolean = false
+    var failFast: Boolean = false
```

Comment:
Should `failFast` be dropped for the new plugin? https://github.com/detekt/detekt/issues/2267

Reply:
Sure lets do it in a next PR.
A quick-flag to enable all rules will only be possible when `--all-rules` is implemented - https://github.com/detekt/detekt/issues/2267 then.